website/src/data/navigation.yaml

```diff
@@ -68,6 +68,8 @@
       path: /deployment-examples/cloudflare
     - title: Deploying to Google Cloud with Static Web Apps
       path: /deployment-examples/google-cloud-static-web-apps
+    - title: Deploying to commercetools Connect
+      path: /deployment-examples/commercetools-connect
```

Comment:
Nit: should it be one of the first options if we want to "promote" Connect more?